FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/Models/FhirToDataLakeProcessingJobResult.cs

```diff
@@ -35,6 +35,16 @@ public class FhirToDataLakeProcessingJobResult
         /// </summary>
         public Dictionary<string, int> ProcessedCount { get; set; } = new Dictionary<string, int>();
 
+        /// <summary>
+        /// Processed count in total.
```

Comment:
it is a little comfused. how about adding information in the summary text that the processed count is for schema type instead of resource type?

Reply:
Add comments here, the variable describes output resource counts.